.github/ISSUE_TEMPLATE/rollout-issue.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Rollout issue
+about: 'Create issue for arcade-services rollout'
+title: Rollout DD-MM-YYYY
+labels: Rollout
+assignees: ''
+
+---
+
+# Purpose
+
+This issue tracks the `arcade-services` repository rollout. On top of the [Rollout](https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/831/Rollout) instructions described on the wiki, it provides the person responsible for the rollout checklist of the steps that should to be performed to rollout services in this repository. All relevant information, including the rollout PR, issues encountered during the rollout and steps taken to resolve them should be linked or added to this issue to keep full audit trail of changes rolled out to production.
+
+# Process
+
+## Build status check (Monday)
+- [ ] Check the status of the [dotnet-arcade-services-weekly](https://dev.azure.com/dnceng/internal/_build?definitionId=993) pipeline
```

Comment:
```suggestion
- [ ] Check the status of the [dotnet-arcade-services-weekly](https://dev.azure.com/dnceng/internal/_build?definitionId=993) pipeline. Rotate any secrets that need manual rotation.
```

Reply:
Added as separate check as it's another well-defined activity